matplotlib/src/MatPlotLibPanel.tsx

```diff
@@ -96,10 +95,12 @@ export const MatPlotLibPanel = (props: MatPlotLibPanelProps): JSX.Element => {
 
   return (
     <div className="mat-plot-lib-panel">
-      {imageSrc && <img src={imageSrc} alt="MatPlotLib render" />}
+      {imageSrc !== undefined && imageSrc !== '' && (
```

Comment:
Kind of unnecessary to check `!== ''`, there's nowhere that we set an empty string to `imageSrc`

Reply:
True. I was just adding it to keep the previous behavior when addressing the strict boolean expressions rule